ydb/core/statistics/ut/ut_aggregator.cpp

```diff
@@ -50,6 +51,42 @@ void CreateUniformTable(TTestEnv& env, const TString& databaseName, const TStrin
     UNIT_ASSERT_C(result.IsSuccess(), result.GetIssues().ToString());
 }
 
+void CreateColumnStoreTable(TTestEnv& env, const TString& databaseName, const TString& tableName) {
+    TTableClient client(env.GetDriver());
+    auto session = client.CreateSession().GetValueSync().GetSession();
+
+    auto fullTableName = Sprintf("Root/%s/%s", databaseName.c_str(), tableName.c_str());
+    auto result = session.ExecuteSchemeQuery(Sprintf(R"(
+        CREATE TABLE `%s` (
+            Key Uint64 NOT NULL,
+            Value Uint64,
+            PRIMARY KEY (Key)
+        )
+        PARTITION BY HASH(Key)
+        WITH (
+            STORE = COLUMN,
+            AUTO_PARTITIONING_MIN_PARTITIONS_COUNT = 10
```

Comment:
`10` should be a function parameter. This let you easy write new tests with another partition count.